php/php-rr-sdk.md

```diff
@@ -466,13 +446,13 @@ the explicit yield required.
 ```php
 public function run(string $input): string
 {
-    $session = new Session([
-        Session::START_TO_FINISH_TIMEOUT => 300,
+    $session = Workflow::newSession([
+        Session::START_TO_FINISH => 300,
         // ... 
     ]);
     
-    $a1 = $session->executeActivity(new ExecuteActivity('activity1'));
-    $a2 = $session->executeActivity(new ExecuteActivity('activity2'));
+    $a1 = $session->executeActivity(Workflow::executeActivity('activity1', ...));
+    $a2 = $session->executeActivity(Workflow::executeActivity('activity2', ...));
```

Comment:
$session->executeActivity('activity2', ...);

Reply:
Both actually should work in my opinion.